services/validation/internal/model/node.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+// Node represents a node stored in the index.
+type Node struct {
+	// ProfileURL is the URL of the node's profile.
+	ProfileURL string `json:"profile_url"`
+	// Version is the version vector of the node.
+	// https://en.wikipedia.org/wiki/Version_vector
```

Comment:
@maxsarratt I believe this is how we are using the Version field. Please let me know if not.

Reply:
You are right 👍 

It is used for preventing concurrency issues in distributed systems.